tests/phpt/openssl/9_openssl_random_pseudo_bytes.php

```diff
@@ -0,0 +1,12 @@
+@ok
+<?php
+
+function test_openssl_random_pseudo_bytes() {
```

Comment:
how to use `for` and the range from negative to positive number?

Reply:
Maybe it worths doing `for` with positive numbers

Reply:
Please check the current solution